.env

```diff
@@ -3,6 +3,7 @@ STUB_STATUS_GO=0
 ETHEREUM_DEV_CLUSTER=1
 MAINNET_WARNING_ENABLED=0
 OFFLINE_INBOX_ENABLED=1
+RPC_NETWORKS_ONLY 1
```

Comment:
@mandrigin minor nit, any reason this line doesn't use the `=` sign like the rest?

Reply:
Seems like typo. When `RPC_NETWORKS_ONLY ` setting is not in `env` file, default value is still 1, that's why it's working even with this typo. /cc @mandrigin